Formula/arpack.rb

```diff
@@ -18,17 +18,19 @@ class Arpack < Formula
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
 
+  depends_on "eigen"
   depends_on "gcc" # for gfortran
   depends_on "open-mpi"
-  depends_on "veclibfort"
+  depends_on "openblas"
 
   def install
     args = %W[
       --disable-dependency-tracking
       --prefix=#{libexec}
-      --with-blas=-L#{Formula["veclibfort"].opt_lib}\ -lvecLibFort
-      F77=mpif77
-      -enable-mpi
+      --with-blas=-L#{Formula["openblas"].opt_lib}\ -lopenblas
+      --enable-mpi
+      --enable-icb
+      --enable-icb-exmm
```

Comment:
icb = ISO_C_binding, allowing linking from C and C++. No down-side AFAICT to enabling this.

explicitly setting `F77=mpif77` appears unneeded.

Reply:
scratch that... F77=mpif77 appears needed after all